sdk/python/README.md

```diff
@@ -9,7 +9,8 @@ Python 2.7 and 3.5+
 ### pip install
 
 ```sh
-pip install kubeflow-training
```

Comment:
I think the plan is to have this package avialable in pypi but I am not the original process. Jinchi owned this part.

Reply:
Hmm this is in the README though so users would expect this to just work. We should only install from PyPI directly when it’s available.